File: C#/BluffinMuffin.Protocol/Lobby/RegisteredMode/AuthenticateUserResponse.cs
namespace BluffinMuffin.Protocol.Lobby.RegisteredMode
{
    /// <summary>
    /// 
    /// </summary>
    public class AuthenticateUserResponse : AbstractBluffinReponse<AuthenticateUserCommand>
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="command"></param>
        public AuthenticateUserResponse(AuthenticateUserCommand command)
            : base(command)
        {
        }
    }
}

File: Python/BluffinMuffin.Protocol/lobby/registered_mode/authenticate_user_response.py
from abstract_response import AbstractResponse
from lobby.registered_mode.authenticate_user_command import AuthenticateUserCommand

__author__ = 'ericmas001@gmail.com'



class AuthenticateUserResponse(AbstractResponse):
    def __init__(self, obj):
        super().__init__(obj, AuthenticateUserCommand(obj['Command']))